jormungandr/src/network/mod.rs

```diff
@@ -122,13 +118,22 @@ impl error::Error for ListenError {
     }
 }
 
+#[derive(Error, Debug)]
+enum PropagateError {
+    #[error("Error sending message to task due to {0}")]
+    InternalCommSend(#[from] futures::channel::mpsc::SendError),
+    #[error("Error receving message from task due to {0}")]
+    InternalCommRecv(#[from] crate::intercom::Error),
+}
```

Comment:
I'm not sure on this, but other parts of the code deal with internal comm errors in the same way (e.g. https://github.com/input-output-hk/jormungandr/blob/79d459f2d25895268fcfce9900cb886e92f9ba62/jormungandr/src/blockchain/process.rs#L391). I just added an error type to avoid having to do error conversion / display implementation each time.